.prettierignore

```diff
@@ -1,8 +1,17 @@
+**/*.rs
 **/*.min.*
+
 /web/book/theme/highlight.js
+/web/playground/build
+/web/website/public
+/web/book/book
```

Comment:
Modern versions of prettier already respect `.gitignore`s IIRC. I guess no problem with having more here, but it shouldn't be required